src/services/pool/decorators/pool.decorator.ts

```diff
@@ -69,6 +70,11 @@ export class PoolDecorator {
       poolService.setFeesSnapshot(poolSnapshot);
       poolService.setVolumeSnapshot(poolSnapshot);
       await poolService.setLinearPools();
+      if (sdkPool) {
+        pool.totalLiquidity = await sdkPools.liquidity(sdkPool);
```

Comment:
As Jarek said, the SDK function can handle a pool even if the schema is not the exact same. You might be able to just do:
```suggestion
        pool.totalLiquidity = await sdkPools.liquidity(poolService.pool);
```